tests/modules/images/regenerate-existing-images/load-tests.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Tests for regenerate-existing-images module load.php.
+ *
+ * @package performance-lab
+ * @group regenerate-existing-images
+ */
+
+class Regenerate_Existing_Images_Load_Test extends WP_UnitTestCase {
+
+	/**
+	 * Test that background_job taxonomy is present in system.
+	 *
+	 * @return void
+	 */
+	public function test_background_job_taxonomy_exists() {
+		$taxonomies = get_taxonomies();
+
+		$this->assertIsArray( $taxonomies );
+		$this->assertContains( 'background_job', $taxonomies );
+	}
+
+	/**
+	 * Ensure taxonomy is non public and would not show in REST.
+	 *
+	 * @return void
+	 */
+	public function test_background_job_taxonomy_is_non_public() {
+		$job_tax = get_taxonomy( 'background_job' );
+
+		$this->assertFalse( $job_tax->public );
+		$this->assertFalse( $job_tax->show_in_rest );
+		$this->assertFalse( $job_tax->hierarchical );
+		$this->assertFalse( $job_tax->show_in_menu );
```

Comment:
I believe this is incorrect, `show_in_menu` is expected to be `true` since `show_ui` is `true`. On that note, please add the assertion for `show_ui` as well.

I just realized that tests aren't running, probably because of the new feature branch. I'll configure this shortly.